### PR TITLE
doc: Add radio propagation

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

Add Radio Propagation tutorial for The Things Advanced + Network Management

#### Screenshots

image

#### Changes

- new document
- ...

#### Notes for Reviewers

@benolayinka Could you please provide new images for each type of propagation based on the following document?
https://docs.google.com/document/d/154WxryZxFyeRNCf03HaqPHC34rLmC_jlHx3UAXNUZeA/edit?usp=sharing

This document is mainly based on:
[TTN Webcast | Radio Theory Crash Course - YouTube](https://www.youtube.com/watch?v=1ZUvyg5zaDg)

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by jpmeijers** [comment]
`Reflections are often allowed to get signals even in dense urban environments.` - the wording in this sentence is a little strange. Maybe rephrase?

**Comment by pradeeka7** [comment]
> `Reflections are often allowed to get signals even in dense urban environments.` - the wording in this sentence is a little strange. Maybe rephrase?

Thanks. will rephrase.

I have rephrased it as:
PS- Reflections help to receive signals even in dense urban environments.

File: doc/content/lorawan/radio-propagation/_index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+section: Additional Information
+description: ""
+weight:
+---
+
+Radio propagation describes how the radio signals travel from the transmitter to the receiver inside the earth's atmosphere or free space.
+
+## Line of Sight
+
+Line of Sight (LoS) is a type of propagation that can transmit and receive data only where the transmitter and receiver are in view of each other without any obstacles between them. Even in the LoS, the signal strength gets reduced when it passes through the air for a long distance.
```
Comment (nejraselimovic) [inline doc/content/lorawan/radio-propagation/_index.md:12]: The first sentence is not really correct. LoS propagation is a type of propagation where signal waves travel along direct transmitter-receiver path without any obstacles. This definition doesn't have anything to do with transceiver's ability to transmit and receive data. Some transceivers can transmit/receive only if LoS exists, while others can do it if LoS doesn't exist, i.e. if there's obstacles between them. 

This part "a type of propagation that can transmit and receive data ..." is also not correct - the propagation cannot transmit or receive anything.

File: doc/content/lorawan/radio-propagation/_index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Attenuation
+
+Attenuation is the loss of signal strength during transmission because of the absorption by various obstacles/materials.
```
Comment (nejraselimovic) [inline doc/content/lorawan/radio-propagation/_index.md:16]: Not necessarily absorption, there is a bunch of reasons, for example reflection. Please write that absorption is just one of possible causes.

File: doc/content/lorawan/radio-propagation/_index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Here is some approximate attenuation at 900 MHz:
+
+- Glass: 2dB
+- Brick: 5dB
+- Masonry brick: 12dB
+- Reinforced concrete: 27dB
+
+## Knife-edge Diffraction
+
+This type of propagation happens a lot in urban environments. A building or a mountain may block the transmitter and receiver antennas creating a shadow zone. These buildings and mountains act as knife edges. When the signal passes over the knife-edge at the top of the building or mountain it is diffracted into the shadow zone. This will usually provide spotty coverage, however, this is important to urban propagation to offer more coverage.
```
Comment (nejraselimovic) [inline doc/content/lorawan/radio-propagation/_index.md:27]:
```suggestion
This type of propagation happens a lot in urban environments. A building or a mountain may block the transmitter and receiver antennas, creating a shadow zone. These buildings and mountains act as knife edges. When the signal passes over the knife-edge at the top of the building or mountain, it is diffracted into the shadow zone. This will usually provide spotty coverage, however, this is important to urban propagation to offer more coverage.
```

File: doc/content/lorawan/radio-propagation/_index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Tropo-scatter
+
+Air in the troposphere contains enough particles up to 10 km. They reliably scatter (throw in various random directions) a small number of radio signals back down to earth. For example, with the scattering, a 900 MHz signal can reach up to 800 km away which is beyond the radio horizon.
```
Comment (nejraselimovic) [inline doc/content/lorawan/radio-propagation/_index.md:31]: Enough particles for what?

File: doc/content/lorawan/radio-propagation/_index.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Reflection
+
+Cities are a good example of such an environment. Obstacles like buildings reflect signals and create multiple paths for the signals to follow. Buildings made out of good conductors reflect nearly all the energy of the signals. However, insulators such as glass, brick walls, and concrete, reflect less energy, allowing the remaining energy to penetrate through. Reflections are often allowed to get signals even in dense urban environments. As radio waves are reflected, they will create both constructive and destructive interference.
```
Comment (nejraselimovic) [inline doc/content/lorawan/radio-propagation/_index.md:35]: Cities are a good example of which environment?

Obstacles reflect signals and create multiple paths for the signals to follow? What does this mean? I think you want to describe multipath fading here, but this just doesn't make any sense. What exactly are those paths that signals are following?

Reflections are often allowed to get signals..? What does this even mean?

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

**Comment by pradeeka7** [comment]
I withdraw this pull request.
